### PR TITLE
Gligars is now Lycanium Z

### PR DESCRIPTION
**Pull request by ghost**
gah so many commits. Sorry >_>

Anyways. Everything works and was rigorously tested.

### CODE REVIEW

File: mods/sgssb/abilities.js [hunk 1]
```diff
@@ -230,4 +230,36 @@ exports.BattleAbilities = {
...
 		name: "node bot",
 	},
+
+	//In Tandem with Lycanium Z's Wreck Havoc Move
+	virus: {
+		shortDesc: "Transforms the pokemon to unown and other odd effects.",
+		id: "virus",
+		name: "Virus",
+		onAfterDamage: function (damage, target, source, move) {
+			if (source && source !== target && move && move.flags['contact']) {
```
Comment (DeathlyPlays) [inline mods/sgssb/abilities.js]: This has the ``move.flags['contact']`` meaning it'll only work if the target makes contact with the user, might want to add that in the descriptions

Comment (ghost) [reply mods/sgssb/abilities.js]: Good idea

File: data/sgssb-data.txt [hunk 2]
```diff
@@ -23,16 +23,18 @@ Signature Move: Mystic Mirage
 Notes: idk
 Rank: N/A.
 
-Gligars
-(Gligar)
-@ Eviolite
-EVs: 172 Atk / 84 Def / 252 SpD
+Lycanium Z
+(Lycanroc)
+@ Lycanium Z
```
Comment (HoeenCoder) [inline data/sgssb-data.txt]: Isn't this going to be a real Z crystal in a few months? We shouldn't override the data of existing (or soon to do existing) moves/items/ect.

File: mods/sgssb/moves.js [hunk 3]
```diff
@@ -568,36 +568,57 @@ exports.BattleMovedex = {
...
+		desc: "Hits 10 times. Confuses the target after.",
+		id: "altstorm",
 		isNonStandard: true,
-		name: "Daredevil",
+		name: "Alt Storm",
 		pp: 10,
 		priority: 0,
 		flags: {protect: 1, mirror: 1},
-		secondary: false,
+		multihit: 10,
```
Comment (HoeenCoder) [inline mods/sgssb/moves.js]: 1 This may not work
2 This is really OP

Comment (ghost) [reply mods/sgssb/moves.js]: 1. It works

2. How about 3/5?

Comment (HoeenCoder) [reply mods/sgssb/moves.js]: that works

File: data/sgssb-data.txt (hunk 2 above)
Comment (ghost) [reply data/sgssb-data.txt]: Not the lycanium z i swear

**Comment by ghost** [comment]
Of course if anything becomes more op than we thought, we can always debuff in a later patch

Idk what else should be fixed tbh

**Comment by ghost** [comment]
Finished (for real this time)

**Pull request merged**